common/styles/scss/_recordset.scss

```diff
@@ -539,3 +544,7 @@ faceting {
 .selected-rows {
     min-height: 45px;
 }
+
+#overflow-container-facet .chaise-checkbox {
+    z-index: 0;
```

Comment:
I think you've added this because of the z-index:1 that is added to checkboxes in app.scss:587, but I would say the better change is removing that rule. I don't think we need to define a z-index on checkboxes.